app/models/setting.rb

```diff
@@ -75,6 +76,12 @@ def self.live_descendants
     disabled_plugins.order_by(:full_name)
   end
 
+  def self.replace_keywords(keyword)
+    if keyword.respond_to?(:gsub)
```

Comment:
Probably check for string, looks cleaner.

Reply:
How should we handle version upgrades, would the user need to update the Setting each time?

Maybe for simplicity the setting should just set whether to display the version on Login page or not.

As for QE tests that expect the version, can we pass an ENV variable also to decide about displaying it or not to make life easier?

Reply:
discussed it with @lzap and @avitova offline and seem that it would work on upgrades,
so you can ignore my previous comment :)